app/ts/background/settings.ts

```diff
@@ -82,7 +82,6 @@ export const defaultRpcs: RpcEntries = [
 		minimized: true,
 		weth: 0x105083929bf9bb22c26cb1777ec92661170d4285n,
 	},
-	/*
 	{
 		name: 'Eth (geth-multi)',
```

Comment:
Can we rename this?  Perhaps `Ethereum (experimental geth)` so it is clear to users they probably shouldn't use it right now?